admin/rules.md

```diff
@@ -313,8 +313,12 @@ layout: default
     2.  A member of the management committee must be a fully paid member of the
         association.
     3.  At each annual general meeting of the association, the members of the
-        management committee must retire from office, but are eligible, on
-        nomination, for re-election.
+        management committee agree to retire from office at a date no more than 
+	two months after the AGM.
+	During this time the outgoing management committee members will:
```

Comment:
The thing with the rules is that they are rules, not guidelines or shoulds or procedure etc. Putting in soft things like these dotpoints is not a problem, but we've rarely found it to add value.

What this section needs is more clear rules about how handover happens, ie for 2 months you have 2 exec, who's actually in charge? are there milestones of handover? 

for example, does the old treasurer only step down once the audit is complete? secretary once the return of association is submitted? that would be better for people leaving big jobs for the next, and helps train new people on that responsibility.

It's taken us 12 months and we still havent solved all handover issues, i'm not sure how to solve handover when its every year and it takes that long to do. we can get better at it but some things are incredibly painful.